latex/programming_interface.tex

```diff
@@ -3636,11 +3636,20 @@ \subsection{Stream class interface}
 \startTable{Stream manipulator}
 \addFootNotes{Manipulators supported by the \codeinline{stream} class}{table.manipulators.stream}
 \addRow
+{
+  flush
+}
+{
+  Triggers a flush operation, that synchronizes the work item stream buffer with the global
+  stream buffer, and that empties the work item stream buffer.  After a flush, the full
+  \codeinline{workItemBufferSize} is available again for subsequent streaming within the work item.
+}
+\addRow
 {
   endl
 }
 {
-  Outputs a new-line character.
+  Outputs a new-line character and then generates a flush.
```

Comment:
I've defined it near the beginning of the section, before the table, with the latest commit.

Reply:
This looks good now thanks, though we can probably simplify the wording of the `flush` manipulator since it's defined earlier.

Reply:
I did try simplifying the flush manipulator description, but then readded because I thought it was too hard to understand with the distributed wording.  I don't really mind either way - do you want me to remove the description from the table?